src/git.js

```diff
@@ -0,0 +1,36 @@
+const childProcess = require('child_process')
+
+const SEPARATOR = '/////'
+const GIT_FORMAT_SPEC = `--pretty=format:"%an${SEPARATOR}%h${SEPARATOR}%s"`
+
+function getGitLogOutput (tagPrefix) {
+  const commitListSpec = `$(git tag | grep '^${tagPrefix}' | tail -n2 | head -n1)..HEAD`
+  const gitCommand = `git log ${commitListSpec} ${GIT_FORMAT_SPEC}`
+
+  return new Promise((resolve, reject) => {
+    childProcess.exec(gitCommand, (error, stdout, stderr) => {
+      if (error) {
+        console.error(error)
+        console.error(stderr)
+        reject(new Error('Git command failed'))
```

Comment:
why not wrap the current error in the `new Error` object?

Reply:
I'm not aware of a standard / conventional way to retain the original error in the thrown error. If you're thinking about something specific, please let me know.
Doing it this way will make the scenario of the failure easy to understand when reading the logs.